runtime/doc/vim_diff.txt

```diff
@@ -119,10 +119,11 @@ of these in your config by simply removing the mapping, e.g. ":unmap Y".
 	nnoremap <C-L> <Cmd>nohlsearch<Bar>diffupdate<Bar>normal! <C-L><CR>
 	inoremap <C-U> <C-G>u<C-U>
 	inoremap <C-W> <C-G>u<C-W>
-	xnoremap * y/\V<C-R>"<CR>
-	xnoremap # y?\V<C-R>"<CR>
 	nnoremap & :&&<CR>
 <
+Default mappings composed of Lua code are not listed above. Use ":map" to see
+the |map-listing|.
```

Comment:
For cases where listing the exact mapping is too much, we can link to the defaults explicitly here. It's useful to have a single place where all the default mappings are linked from.

```
|v_#-default|
|v_star-default|
```